src/main/java/cs361/battleships/models/Board.java

```diff
@@ -39,34 +41,68 @@ public boolean placeShip(Ship ship, int x, char y, boolean isVertical) {
 
 		// Calculate the location of the ship based off of type, starting location, and verticality
 		if (isVertical) {
-			if (x + ship.getLength() < 11 && x > 0) {
+			if (x + ship.getLength() < 11 && x > 0 && !isSubmarine) {
 				for (int i = 0; i < ship.getLength(); i++) {
 					shipLoc.add(new Square(x + i, y));
 				}
+			} else if (isSubmarine && x > 0 && x + ship.getLength()-1 <= 11 && y + 2 - 'A' < 11) {
+				for (int i = 0; i < ship.getLength() - 1; i++) {
+					shipLoc.add(new Square(x + i, y));
+				}
+				shipLoc.add(new Square(x + 2, (char)(y + 1)));
 			} else {
 				return false;
 			}
 		} else {
-			if (y + ship.getLength() - 'A' < 11 && y >= 'A') {
+			if (y + ship.getLength() - 'A' < 11 && y >= 'A' && !isSubmarine) {
 				for (int i = 0; i < ship.getLength(); i++) {
 					shipLoc.add(new Square(x, (char)(y + i)));
 				}
+			} else if (isSubmarine && x > 1 && y + ship.getLength()-1 - 'A' < 11) {
+				for (int i = 0; i < ship.getLength() - 1; i++) {
+					shipLoc.add(new Square(x, (char)(y + i)));
+				}
+				shipLoc.add(new Square(x - 1, (char)(y + 2)));
 			} else {
 				return false;
 			}
 		}
 
-		// Check if there is already a ship in this location
-		for (Square locToCheck : shipLoc) {
-			for (Ship shipToCheck : placedShips) {
-				for (Square shipCheckLocToCheck : shipToCheck.getOccupiedSquares()) {
-					if (locToCheck.getRow() == shipCheckLocToCheck.getRow() && locToCheck.getColumn() == shipCheckLocToCheck.getColumn()) {
-						return false;
+		// If sub, then check and mark as submerged
+		if (isSubmarine) {
+			for (Square locToCheck : shipLoc) {
+				for (Ship shipToCheck : placedShips) {
+					for (Square shipCheckLocToCheck : shipToCheck.getOccupiedSquares()) {
+						if (locToCheck.getRow() == shipCheckLocToCheck.getRow() && locToCheck.getColumn() == shipCheckLocToCheck.getColumn()) {
+							sub_submerged = true;
+							break;
+						}
+					}
+				}
+			}
+		} else {
+			// Check if there is already a ship in this location
+			// Check for submarine, if submarine is in square, then mark it as submerged
+			for (Square locToCheck : shipLoc) {
+				for (Ship shipToCheck : placedShips) {
+					if (!shipToCheck.getKind().equals("SUBMARINE")) {
```

Comment:
Creating special conditions isn't generally the best due to object oriented design, but theres really no way to make it more scalable here so its fine.